locales/en/app.json

```diff
@@ -22,6 +22,7 @@
   "navigation": {
      "general": "to",
      "auth": "Your details",
+     "accessibilityStatement": "Accessibility Statement",
```

Comment:
From content design, needs to be lowercase statement:```suggestion
     "accessibilityStatement": "Accessibility statement",
```

Reply:
incorporated suggestion